### PR TITLE
form processing tweaks

### PR DESCRIPTION
**Pull request by snopoke**
review by commit :tropical_fish:

### CODE REVIEW

File: corehq/form_processor/management/commands/run_submission_reprocessing_queue.py [hunk 1]
```diff
@@ -35,8 +35,12 @@ def get_fetching_interval(self):
     def get_items_to_be_processed(cls, utcnow):
         _record_datadog_metrics()
         queued_threshold = utcnow - timedelta(minutes=ENQUEUING_TIMEOUT)
-        queue_filter = Q(date_queued__isnull=True) | Q(date_queued__lte=queued_threshold)
-        query = UnfinishedSubmissionStub.objects.filter(queue_filter).order_by('timestamp')
+        min_processing_age = datetime.utcnow() - timedelta(minutes=5)
+        filters = Q(date_queued__isnull=True) | Q(date_queued__lte=queued_threshold)
+
+        # wait 5 mins before processing to avoid processing during form submission
```
Comment (jmtroth0) [inline corehq/form_processor/management/commands/run_submission_reprocessing_queue.py]: This seems a bit brittle. What happens if it it's processed during form submission?

**Review by jmtroth0 — APPROVED** [review]
Seems good. Just a couple questions

**Comment by jmtroth0** [comment]
Test errors seem real, although perhaps a bit rude `Exception: mwa ha ha!`

File: corehq/form_processor/reprocess.py [hunk 2]
```diff
@@ -73,12 +73,23 @@ def _perfom_post_save_actions(form, save=True):
     )
     with cache as casedb:
         case_stock_result = SubmissionPost.process_xforms_for_cases([form], casedb)
+        case_models = case_stock_result.case_models
+
+        if interface.use_sql_domain:
```
[the remark this answers is not in the document]
Comment (snopoke) [reply corehq/form_processor/reprocess.py]: yea, couch doesn't publish directly, we have a pillow that pulls the changelog from couch and published to kafka

File: corehq/form_processor/management/commands/run_submission_reprocessing_queue.py (hunk 1 above)
Comment (snopoke) [reply corehq/form_processor/management/commands/run_submission_reprocessing_queue.py]: It doesn't really matter, it's just duplicate work.

**Pull request merged**